operator/main.py

```diff
@@ -293,6 +293,16 @@ def deploy_csi_pods(core_v1_client):
             return
 
     # Deploy CSI Pods
+    api_instance=client.VersionApi().get_code()
+    if api_instance.major>"1" or api_instance.major=="1" and api_instance.minor>="14":
+        filename = os.path.join(MANIFESTS_DIR, "csi-driver-object.yaml")
+        template(filename, namespace=NAMESPACE, kadalu_version=VERSION)
+        execute(KUBECTL_CMD, "create", "-f", filename)
+    else:
```

Comment:
btw why kadalu require minimum  1.14  as stated here? https://github.com/kadalu/kadalu#pre-requisites

@amarts

Reply:
5hat requirement came from what we tested.... We havent tested for version below that.

Reply:
if the minimum version is 1.14 i can remove if else check

Reply:
@aravindavk you have any idea about why it is done this way? different between object / crd yml?

Reply:
@amarts more info here https://kubernetes-csi.github.io/docs/csi-driver-object.html